verifier/verifier.go

```diff
@@ -119,11 +122,14 @@ func (v *verifier) Verify(ctx context.Context, desc ocispec.Descriptor, signatur
 	payload := &envelope.Payload{}
 	err = json.Unmarshal(outcome.EnvelopeContent.Payload.Content, payload)
 	if err != nil {
+		logger.Debug("Failed to unmarshal the payload content in the signature blob to envelope.Payload")
```

Comment:
Should it be an error level warning?
```suggestion
		logger.Error("Failed to unmarshal the payload content of the signature envelope")
```

Reply:
Nice catch!